src/roswire/ros2/state.py

```diff
@@ -28,31 +28,61 @@ class ROS2SystemState(SystemState):
         A mapping from topics to the names of subscribers to that topic.
     services: Mapping[str, Collection[str]]
         A mapping from services to the names of providers of that service.
+    clients: Mapping[str, Collection[str]]
+        A mapping from services to the names of clients of that service.
+    action_servers: Mapping[str, Collection[str]]
+        A mapping from actions to the names of providesr of that action.
+    action_clients: Mapping[str, Collection[str]]
+        A mapping from actions to the names of clients of that action
     nodes: AbstractSet[str]
         The names of all known nodes running on the system.
     topics: AbstractSet[str]
         The names of all known topics on the system with at least one
         publisher or one subscriber.
+    all_services: AbstractSet[str]
+        The name of all the known services on the system with at least
+        one server or client
+    actions: AbstractSet[str]
```

Comment:
Then this would become `action_names`, I suppose?

Reply:
Done